infra/base-images/base-runner/Dockerfile

```diff
@@ -63,3 +64,15 @@ ENV MSAN_OPTIONS="print_stats=1:strip_path_prefix=/workspace/:symbolize=1"
 ENV UBSAN_OPTIONS="print_stacktrace=1:print_summary=1:silence_unsigned_overflow=1:strip_path_prefix=/workspace/:symbolize=1"
 ENV FUZZER_ARGS="-rss_limit_mb=2560 -timeout=25"
 ENV AFL_FUZZER_ARGS="-m none"
+
+# Download and install the latest stable Go.
```

Comment:
Copied from base-builder

Maybe we can reuse the binaries already present in /out/root/go...

Reply:
so we need golang to be installed in the base-runner image, as it's being used for generating a coverage report?

Reply:
@oliverchang as we also install Go in the base-runner, would you be ok with us doing that once in the `base-image` instead?

Reply:
Leaving it here is probably fine. 

We plan to diverge images here and migrate runners to be based on ClusterFuzz images instead, so we'd have to add it back here anyway.

Reply:
ack, thanks Oliver!